src/hipscat_import/__init__.py

```diff
@@ -4,4 +4,17 @@
 from .command_line_arguments import parse_command_line
 from .file_readers import CsvReader, ParquetReader, fits_reader, get_file_reader
 from .map_reduce import map_to_pixels, reduce_pixel_shards
+from .resume_files import (
+    clean_resume_files,
```

Comment:
Any reason not to just import * at this point? I know there are a few private type functions in those files, but you are already using the _ prefix to indicate those should be treated as internal only.